tests/Keboola/DbExtractor/AbstractSnowflakeTest.php

```diff
@@ -1,5 +1,8 @@
 <?php
-namespace Keboola\Test;
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractor\Tests;
```

Comment:
Tests do not need to be moved to subdirectory. It would be enough to set:

```
"autoload-dev": {
        "psr-4": {
            "Keboola\\DbExtractor\\Tests\\": "tests/"
        }
    },
```

Also, ideally the namespace should also be different (e.g. `Keboola\DbExtractor\SnowflakeExtractor`), but I think that it's hardcoded in the common extractor... so that's fine. The note about test paths still stands though.